californium-core/src/main/java/org/eclipse/californium/core/observe/Observation.java

```diff
@@ -52,6 +52,20 @@ public Request getRequest() {
 	}
 
 	/**
+	 * @return a cloned observation with a shallow clone of request
+	 */
+	public Observation cloneObservation() {
```

Comment:
Ok, thinking about it I also came to the conclusion that it is probably not a good idea to make `Observation` cloneable in general since in a real implementation we would actually create the new instance by means of unmarshalling from persistent state.